### ISSUE TITLE
organization export not showing actual data

### ISSUE THREAD

**Issue by lawsangel** [osTicket/osTicket#4822]
### Prerequisites

* [? ] Can you reproduce the problem in a fresh installation of the "develop" branch?
* [no] Do you have any errors in the PHP error log, or javascript console?
* [Yes] Did you check the [osTicket forums](http://osticket.com/forum/categories)?
* [yes] Did you [perform a cursory search](https://github.com/osTicket/osTicket/issues) to see if your bug or enhancement is already reported?

For more information on how to write a good [bug report](https://github.com/osTicket/osTicket/wiki/Github-Issues-Guidelines)

### Description

When i click the export ticket option on organizations, the exel document doesnt show the subject, from, email, priority, status, instead it shows the following.
cdata__subject
user__name
user__emails__address
cdata__priority
status__id

### Steps to Reproduce

Click export when on the tickets tab in orginisations.
click export.
view excel document

**Expected behavior:** [Document should have the actual username, email address, status etc]

**Actual behavior:** [Showing cdata__subject, user__name, user__emails__address etc instead]

### Versions
v1.11 (d4e240b) (upgraded from v1.10)
Web Server Software | Microsoft-IIS/8.5
MySQL Version | 10.3.12
PHP Version | 7.2.7
Windows Server 2012 R"

Admin panel -> Dashboard -> Information which also additionally gives you information about your server.

Also, please include the OS and what version of the OS you're running. As well as your browser and browser version.

**Comment by ntozier** [comment]
confirmed.

**Comment by JediKev** [comment]
This is a known bug that we are working on. The exporter for both Users and Orgs need to be refactored.

Cheers.

**Comment by lawsangel** [comment]
Hi
Do i need to keep checking back here for an update, or should i be looking somewhere else?
Sorry i'm not familiar with how it works here.
Thanks

**Comment by lawsangel** [comment]
Is there any word on this?
I've not heard back from anyone on here or the OSTicket forums.
I upgraded to the newest version 1.12 today and it still doesn't work.
I'm at the point of dumping OSTicket now.

**Comment by JediKev** [comment]
@lawsangel 

If there was any word you'd see it here. With this being said, it is still a bug that is actively being looked into.

Cheers.

**Comment by JediKev** [comment]
@lawsangel

Here you go:
#4880

Cheers.

**Comment by lawsangel** [comment]
Thanks
I've looked at it but i'm not sure what i'm suppose to do.
Do i just edit both files in question manually?
I've tried editing both files with Notepad++ (Disabled osticket via IIS first)
Started OSticket again, but still the same results.

**Comment by JediKev** [comment]
It seems like you have it installed from a zip meaning it’s not a git clone or deployed from a git clone. You will have to apply the changes manually in this case.

Cheers.

**Comment by lawsangel** [comment]
So thats what i've done as stated above, edited manually with notepad++
It still doesnt work.

**Comment by lawsangel** [comment]
Appologies i take that back.
Copying and pasting didnt include some of the gap lines, i've just put them in manually and it's now working.
Thanks for doing that.
Will i have to do that if i install it again, or will that now be added to future packages?

**Comment by JediKev** [comment]
@lawsangel 

It will be included in later maintenance releases/major releases once approved and merged.

Cheers.

**Comment by lawsangel** [comment]
Just noticed that it's working for the organisations export, but not the user.

**Comment by JediKev** [comment]
@lawsangel 

Right...read the pull request. It's only for Organizations. Users are being worked on as well.

Cheers.

**Comment by lawsangel** [comment]
Sorry, just assumed it would fix both as it's the same issue.

**Comment by JediKev** [comment]
@lawsangel 

Updated to address both issues now.

Cheers.

**Comment by lawsangel** [comment]
Excellent, thats it now working.
Kind regards
